finalPrep/README.md

```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+`This will definitely change when I have a better understanding of standard weekly group work. I will be able to get more granular with homework v. project v. study time. And I will build in time to meet with collaborators based on everyone's availability.`
+
+- `Add Week 1 Screenshot Here`![](assets/README-826b900a.png)
+
+- `Add Week 2 Screenshot Here`![](assets/README-115c9eef.png)
+
+- `Add Week 3 Screenshot Here`![](assets/README-440d1e3b.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist [HERE](https://gist.github.com/kbergsund/446dbb723a3560a61bfcaa51a94e9d39):
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist [HERE](https://gist.github.com/kbergsund/deb2c028794c20802a6049bb49731ae8)
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist [HERE](https://gist.github.com/kbergsund/1bf79941d0092f8f9418e83df7f2a5f3)
```

Comment:
Excellent thoughts here- It is great you are self-aware about some work/life balance struggles associated with sharing a work space with a partner- you'll find coding takes pretty focussed energy at times, so glad you are thinking about how to work with your partner through this.